src/processor/nonce.service.ts

```diff
@@ -53,10 +52,10 @@ export class NonceService implements OnApplicationBootstrap {
     const nonceNumber = (await this.blockchainService.getNonce(this.accountId)).toNumber();
     const keys = this.getNextPossibleKeys(nonceNumber);
     // @ts-ignore
-    const nextNonceIndex = await this.redis.incrementNonce(...keys, keys.length, RedisUtils.NONCE_KEY_EXPIRE_SECONDS);
+    const nextNonceIndex = await this.cacheService.redis.incrementNonce(...keys, keys.length, CacheUtils.NONCE_KEY_EXPIRE_SECONDS);
```

Comment:
These nested calls are worrying me, but perhaps that's a refactor for next PR/future?

Reply:
Yeah--I didn't feel like touching this except to update the call path for the refactor